bool airborne
bool in_mission
bool landing_phase
